### PR TITLE
Milestone 3: Form and footer

### PR DESCRIPTION
**Pull request by DuaneDave**
### Form and Footer Added and Styled

 In this milestone we:

   - added a contact form 
   - added some validations on the form
   - wired up the form submit to formspree
   - added the footer and its icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -258,5 +258,22 @@ <h2>Skills</h2>
...
+    <footer>
+      <section class="contact flex">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form class="flex" action="https://formspree.io/f/xdobralq" method="post"
+        >
+          <input name="fname" type="text" required placeholder="Full Name" maxlength="30">
+          <input name="email" type="text" required placeholder="Email address">
+          <textarea name="message" placeholder="Enter text here" maxlength="500"></textarea>
```
Comment (zieeco) [inline index.html:270]: - Please, kindly make your Textarea field required to prevent submitting the form when its message field is empty.

Comment (zieeco) [reply index.html:270]: **[OPTIONAL]**

- The footer should only contain the list of social media icons and the contact section should not be wrapped inside of the footer as this was stated in the project requirement, but it is actually not to be added in the footer.  Kindly fix this to improve and enforce best practices in using the html5 semantic tag.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @DuaneDave,


Great job so far! 👏 
You've done well implementing the contact form, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DuaneDave) [reply index.html]: all changes made

**Pull request merged**